packages/jetpack-ai-calypso/src/logo-generator/hooks/use-logo-generator.ts

```diff
@@ -319,9 +319,21 @@ User request:${ prompt }`;
 		debug( 'Generating logo for site', siteId );
 
 		setIsRequestingImage( true );
-		increaseAiAssistantRequestsCount();
 
 		try {
+			if ( ! siteId ) {
+				throw new Error( 'Missing siteId or logo' );
+			}
+
+			const feature = getAiAssistantFeature( siteId );
+			const cost = feature.costs?.[ 'jetpack-ai-logo-generator' ]?.logo;
+
+			if ( cost == null ) {
```

Comment:
Should we use `===` here?

Reply:
That was intentional. `==` matches both `null` and `undefined`.

Reply:
Nice!